src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $borderColor;
+  transition: all $transition-time;
```

Comment:
Don't use `all` for transition property, specify only the needed ones 
As `all` affects in the bad way on the website performance

Reply:
done